### PR TITLE
Add INSTALL.txt file

### PR DESCRIPTION
**Pull request by pubudu91**

### CODE REVIEW

File: resources/tools/INSTALL.txt [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+-------------
+
+Ballerina requires JRE 8 Update 172+.
+It is recommended to use AdoptOpenJDK. Check https://adoptopenjdk.net/releases.html to download the JRE.
+
+
+INSTALLING BALLERINA
+--------------------
+
+- Unzip the Ballerina archive in the desired directory.
```
Comment (hasithaa) [inline resources/tools/INSTALL.txt]: Shall we say this will be the BALLERINA_HOME directory ?

File: resources/tools/INSTALL.txt [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+- Set the path of the installation directory to the BALLERINA_HOME environment variable.
+
+- Add the BALLERINA_HOME environment variable to your system's path variable.
```
Comment (hasithaa) [inline resources/tools/INSTALL.txt]: This has to be %BALLERINA_HOME%\bin for Windows, $BALLERINA_HOME\bin for Unix.

File: resources/tools/bin/ballerina [hunk 3]
```diff
@@ -26,7 +26,11 @@ then
     JAVA_COMMAND=java
     if test -d "$CURRENT_PATH/../dependencies/jdk8u202-b08-jre"; then
       JAVA_COMMAND="$CURRENT_PATH/../dependencies/jdk8u202-b08-jre/bin/java"
+    elif [[ `$JAVA_COMMAND -version 2>&1 | grep -e "1.8."` = "" ]]; then
```
Comment (Kishanthan) [inline resources/tools/bin/ballerina]: shall we not use -e option with grep? There is a known issue with solaris https://github.com/ballerina-platform/ballerina-distribution/issues/359

File: resources/tools/INSTALL.txt [hunk 4]
```diff
@@ -0,0 +1,46 @@
+                                                    ~ BALLERINA ~
+                                                    =============
+
+
+PREREQUISITES
+-------------
+
+Ballerina requires JRE 8 Update 172+.
+It is recommended to use AdoptOpenJDK. Check https://adoptopenjdk.net/releases.html to download the JRE.
```
Comment (hasithaa) [inline resources/tools/INSTALL.txt]: Shall we say download jre from here? https://github.com/AdoptOpenJDK/openjdk8-binaries/releases/tag/jdk8u202-b08

File: resources/tools/INSTALL.txt [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+- Unzip the Ballerina archive in the desired directory.
+
+- Set the path of the installation directory to the BALLERINA_HOME environment variable.
+
+- Add the BALLERINA_HOME environment variable to your system's path variable.
+
+- If you have installed Java 8 already, you are all set! Just make sure that the JAVA_HOME environment variable is set correctly.
+
+- If you do not have Java 8, or if you have a different version of Java, download a compatible JRE 8 version as specified in the prerequisites.
+	- To set the obtained JRE as the one used by Ballerina, extract the downloaded JRE to the <BALLERINA_HOME>/dependencies/jdk8u202-b08-jre/ directory.
```
Comment (hasithaa) [inline resources/tools/INSTALL.txt]: We can ask them to unzip downloaded jre zip/tar.gz to <BALLERINA_HOME>/dependencies/.  The final folder should be something like this. <BALLERINA_HOME>/dependencies/jdk8u202-b08-jre/

File: resources/tools/INSTALL.txt [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+
```
Comment (Kishanthan) [inline resources/tools/INSTALL.txt]: Once the jre is copied to this directory, the final directory structure would look like

```
$ tree <BALLERINA_HOME>/dependencies/jdk8u202-b08-jre -L 1
/Library/Ballerina/dependencies/jdk8u202-b08-jre
├── ASSEMBLY_EXCEPTION
├── LICENSE
├── THIRD_PARTY_README
├── bin
├── lib
├── man
└── release

3 directories, 4 files

```

File: resources/tools/INSTALL.txt [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+
```
Comment (Kishanthan) [inline resources/tools/INSTALL.txt]: Final step is to verify whether Ballerina is installed properly.

We can use the following command.

```
$ ballerina version                     
jBallerina 1.1.0
Language specification 2019R3
Command 0.8.0
```

**Pull request merged**